### PR TITLE
Add missing char; Replace newer release image; Add needed link

### PR DESCRIPTION
**Pull request by leemyongpakvn**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Newer release image
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Need to be backported to `1.7`

**Comment by leemyongpakvn** [comment]
@NeOMakinG Missing char and Needed link are already back-ported at PrestaShop/docs/pull/1378 . I think we could keep the old release image for 1.7 docs.

**Comment by NeOMakinG** [comment]
> @NeOMakinG Missing char and Needed link are already back-ported at [/pull/1378](https://github.com/PrestaShop/docs/pull/1378) . I think we could keep the old release image for 1.7 docs.

Thanks for the precision

File: basics/installation/localhost.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ PrestaShop comes in two "flavors":
 - **Release package**. A zip package, tuned for production environments.
 - **Development version**. The raw source code as it is on the GitHub repository, including automated test suites, build scripts and source codes for assets that are otherwise compiled (like javascript and css files).
 
-![Download files](../img/download-links.png)
+![Download files](https://user-images.githubusercontent.com/3759923/182030932-c9a26441-41b1-4558-a397-5a936ede216b.png)
```
Comment (NeOMakinG) [inline basics/installation/localhost.md]: Btw, do we really want to rely on a hosted image? I'm curious

Comment (leemyongpakvn) [reply basics/installation/localhost.md]: I saw only 1 option to attach image there, have just uploaded and used an image in img folder.

**Review by NeOMakinG — APPROVED** [review]

**Comment by leemyongpakvn** [comment]
close my mistake: rename branch make it close

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you @leemyongpakvn ! :)

**Comment by leemyongpakvn** [comment]
@kpodemski There is a problem with image link now. Due to the difference between
my branch URL [patch-1/basics/installation/localhost.md](https://github.com/leemyongpakvn/docs/blob/patch-1/basics/installation/localhost.md)
and 
the main repository URL [8/basics/installation/localhost/](https://devdocs.prestashop.com/8/basics/installation/localhost/)
, the image link on my branch img/Prestashop_1.7.8.6_release.png must be changed to ```..```/img/Prestashop_1.7.8.6_release.png on the main repository.

**Pull request merged**